SECURITY.md

```diff
@@ -0,0 +1,52 @@
+Our top priority is keeping our customers' data safe. If you have found an issue in our systems, please reach out to us.
+
+## Reporting a Vulnerability
+
+If you believe you have found a vulnerability, please disclose by contacting us: security@highlight.io
+
+Please try your best to describe a clear and realistic impact for your report.
+
+## Supported versions
+
+| Version     | Supported |
+|-------------|-----------|
+| main branch | ️✅        |
+| any other   | ❌         |
+
+
+## Vulnerabilities we care about 🫣
+> Note: Please use a [self-hosted instance](https://www.highlight.io/docs/getting-started/self-host/self-hosted-hobby-guide) to perform any tests. Do not use the production app.highlight.io product for security testing.
```

Comment:
Is this really fair? Prod uses firebase whereas local dev uses simple auth.

Reply:
yes, since it's possible to configure a test firebase account and provide the necessary environment variables to use firebase with a local deploy.